ui/src/ScholarReader.tsx

```diff
@@ -404,12 +409,41 @@ class ScholarReader extends React.PureComponent<ScholarReaderProps, State> {
           undefined,
           { name: "XYZ" },
           box.left + SCROLL_OFFSET_X,
-          box.top + SCROLL_OFFSET_Y
+          box.top + SCROLL_OFFSET_Y,
+          null,
         ]
       });
     }
   }
 
+  scrollSymbolIntoView() {
```

Comment:
Can you provide a clarifying function-level comment, or rename this function, to indicate that this function will scroll it horizontally into view if it's occluded by a drawer? The current name suggests that this might also do vertical scrolling, or scrolling it rightward into view.